[2025-08-18T13:23:52.177+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-18T13:23:52.282+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: youtube_pipeline_batch_data.run_q_05_tags_core_metrics manual__2025-08-18T13:16:17.589887+00:00 [queued]>
[2025-08-18T13:23:52.309+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: youtube_pipeline_batch_data.run_q_05_tags_core_metrics manual__2025-08-18T13:16:17.589887+00:00 [queued]>
[2025-08-18T13:23:52.310+0000] {taskinstance.py:2303} INFO - Starting attempt 4 of 4
[2025-08-18T13:23:52.345+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): run_q_05_tags_core_metrics> on 2025-08-18 13:16:17.589887+00:00
[2025-08-18T13:23:52.362+0000] {standard_task_runner.py:63} INFO - Started process 4983 to run task
[2025-08-18T13:23:52.374+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'youtube_pipeline_batch_data', 'run_q_05_tags_core_metrics', 'manual__2025-08-18T13:16:17.589887+00:00', '--job-id', '317', '--raw', '--subdir', 'DAGS_FOLDER/youtube_pipeline.py', '--cfg-path', '/tmp/tmpq1mrusvb']
[2025-08-18T13:23:52.378+0000] {standard_task_runner.py:91} INFO - Job 317: Subtask run_q_05_tags_core_metrics
[2025-08-18T13:23:52.489+0000] {task_command.py:426} INFO - Running <TaskInstance: youtube_pipeline_batch_data.run_q_05_tags_core_metrics manual__2025-08-18T13:16:17.589887+00:00 [running]> on host 47205959681c
[2025-08-18T13:23:52.902+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='youtube_pipeline_batch_data' AIRFLOW_CTX_TASK_ID='run_q_05_tags_core_metrics' AIRFLOW_CTX_EXECUTION_DATE='2025-08-18T13:16:17.589887+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-18T13:16:17.589887+00:00'
[2025-08-18T13:23:52.907+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-18T13:23:52.992+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-18T13:23:52.995+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'PGPASSWORD=citus psql -h citus_coordinator -U citus -d youtube_dw -f /opt/***/sql/q_05_tags_core_metrics']
[2025-08-18T13:23:53.026+0000] {subprocess.py:86} INFO - Output:
[2025-08-18T13:23:53.312+0000] {subprocess.py:93} INFO - psql: error: /opt/***/sql/q_05_tags_core_metrics: No such file or directory
[2025-08-18T13:23:53.318+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-08-18T13:23:53.324+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-18T13:23:53.361+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-08-18T13:23:53.373+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=youtube_pipeline_batch_data, task_id=run_q_05_tags_core_metrics, execution_date=20250818T131617, start_date=20250818T132352, end_date=20250818T132353
[2025-08-18T13:23:53.408+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 317 for task run_q_05_tags_core_metrics (Bash command failed. The command returned a non-zero exit code 1.; 4983)
[2025-08-18T13:23:53.447+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-08-18T13:23:53.508+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-18T13:23:53.524+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
